common/constants.go

```diff
@@ -32,19 +33,18 @@ const (
 var Rewound = uint64(0)
 
 var TIP2019Block = big.NewInt(1)
-var TIPSigning = big.NewInt(3000000)
-var TIPRandomize = big.NewInt(3464000)
+var TIPSigning = big.NewInt(225000)
+var TIPRandomize = big.NewInt(225000)
+var TIPV2SwitchBlock = big.NewInt(7074000)
```

Comment:
help me change to 0

Reply:
I prefer deal with all those useless code in a seperate clean up PR